packages/client/src/components/WelcomeBox/Heading/Heading.component.js

```diff
@@ -0,0 +1,14 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export const Heading = ({ title }) => {
```

Comment:
It's not uncommon to have a Heading component since typography is a big part of consistent design, but then I would expect it to live standalone from the WelcomeBox. 
So if this Heading component is able to encapsulate how **all** headings (with props) in the app should look then I would make it a component. 

But, if this is only for one place in the app, in the welcome box then making this its own component feels like overhead. 

Similar for Paragraph.